src/app/components/home/home.component.ts

```diff
@@ -51,6 +56,11 @@ class HomeComponent extends PageComponent implements OnInit {
         () => (this.projectList = List([]))
       );
   }
+
+  public get viewBox(): string {
+    const width = Math.max(70, Math.floor(this.title.length * (157 / 15)));
```

Comment:
Explain constants?

It'd be good if this wasn't a function either. Set when title changes as a basic property so we avoid invoking function on updates